public/index.html

```diff
@@ -13,8 +13,12 @@
   </head>
   <body>
     <main id="app">
-      <div class="hero">
-        <div class="container">
+      <div class="hero"> 
+        <div class="container">   
+            <form style="width: 55%; margin-left: 200px;" v-on:submit.prevent="searchSparkle">
+              <label for="body">/search</label>
+              <input type="text" placeholder="insert username e.g @spongebob" v-model="username" />
+            </form>
```

Comment:
This is fantastic! I like how you used a [form](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) to capture the [submit event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) by clicking the `Enter` key. 

I think the [inline style](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style) is a great place to start. To separate the content from the appearance I would like to suggest we try to extract a [CSS class](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) so that we can tweak the look of the search field from the CSS stylesheet. What do you think?

for example:

```html
<!-- index.html -->
<form class="search" v-on:submit.prevent="searchSparkle">
```

```css
// application.css
form.search {
  width: 55%;
  margin-left: 200px;
}
```